de.metas.adempiere.libero.libero/src/main/java/org/eevolution/callout/PP_Product_BomModuleInterceptor.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * #%L
+ * de.metas.adempiere.libero.libero
+ * %%
+ * Copyright (C) 2019 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package org.eevolution.callout;
+
+import org.adempiere.ad.callout.spi.IProgramaticCalloutProvider;
+import org.adempiere.ad.modelvalidator.AbstractModuleInterceptor;
+import org.adempiere.ad.ui.api.ITabCalloutFactory;
+import org.eevolution.model.I_PP_Product_BOM;
+
+public class PP_Product_BomModuleInterceptor extends AbstractModuleInterceptor
```

Comment:
what i meant to say in https://github.com/metasfresh/metasfresh/pull/6472#pullrequestreview-391963061 is:
* please ditch this class (`PP_Product_BomModuleInterceptor`)
* register the new callout and the new tab-callout in the constructor of the existing model interceptor `org.eevolution.model.validator.PP_Product_BOM`